sports_club/demo/appointment_type.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="appointment_type_1" model="appointment.type">
+        <field name="is_published" eval="True"/>
+    </record>
+    <record id="appointment_type_2" model="appointment.type">
+        <field name="is_published" eval="True"/>
+    </record>
```

Comment:
I think those ones do not need to be published, as it is accessible through an url link, constructed from the `short_code` of the `appointment.invite`